from customerror import CustomError

ErrBadParameters = CustomError("one of the parameter is incorrect")